unit_test/docker_container_test.py

```diff
@@ -1,11 +1,15 @@
+import atexit
+import os
 import platform
 import random
 import shutil
 import subprocess
+import tempfile
 import textwrap
 from pathlib import Path, PurePath, PurePosixPath
 
 import pytest
+import toml  # type: ignore[import]
```

Comment:
You might not need the `ignore[import]`, if you install all the dev-dependencies on your development machine - we have `types-toml` listed there.

Reply:
I'll try removing it, but I think the CI will error. I added it in because mypy told me to.

Reply:
I was able to get around this by adding a line to `.pre-commit-config.yaml` indicating that ` - types-toml` is a mypy dependency.